stable/falco/templates/configmap-rules.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "falco.fullname" . }}-rules
+  labels:
+    app: {{ template "falco.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data: {}
```

Comment:
Why do you create an empty configmap. The above rules file is not used by the chart. Resources created by Helm should not be edited manually as you suggest in the readme. This would get you in trouble when updating a release.

Reply:
Because we want to allow that chart users adds its own rules. Let me to explain a bit more the context. 

Although Falco comes with a nice default rule set for detecting weird behavior in containers, our users are going to customize the run-time security rule sets or policies for the specific container images and applications they run. In Falco we provide an /etc/falco/rules.d directory for dropping new files with rules and we want to allow our users to upload its rules to that directory, so our idea was to mount the /etc/falco/rules.d directory as a configMap and allow users to add rules to that configMap.

For us, the ideal situation would be allowing to pass rules files when installing the helm but 
I have been reading about this problem https://github.com/kubernetes/helm/issues/3276 or in [Accessing files inside templates](https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/accessing_files.md#accessing-files-inside-templates) and looks like this is not supported. Perhaps you have some tips or guidance or a workaround for this issue?

Thanks!

Reply:
If you use Helm you should use it exclusively. Manipulating file installed by Helm will get you in trouble. Why don't you make the configmap configurable via `values.yaml`?